/home/ubuntu/.local/lib/python3.10/site-packages/distributed/node.py:183: UserWarning: Port 8787 is already in use.
Perhaps you already have a cluster running?
Hosting the HTTP server on port 33927 instead
  warnings.warn(
Distributing data time: 4.805498838424683
submit_one_func 0.05686068534851074
submit_one_func 0.0005941390991210938
submit_one_func 0.0004830360412597656
submit_one_func 0.00047469139099121094
submit_one_func 0.00047206878662109375
submit_one_func 0.0004596710205078125
submit_one_func 0.0004611015319824219
submit_one_func 0.0004668235778808594
submit_one_func 0.00044655799865722656
submit_one_func 0.00046944618225097656
submit_one_func 0.0004596710205078125
submit_one_func 0.00046825408935546875
submit_one_func 0.0004706382751464844
submit_one_func 0.0004715919494628906
submit_one_func 0.0004620552062988281
submit_one_func 0.00046181678771972656
Execution time: 0.8647606372833252
Get table content time: 0.18681764602661133
Get table content time: 0.4719841480255127
Get table content time: 0.6424741744995117
Get table content time: 0.4644789695739746
Get table content time: 0.6410157680511475
Get table content time: 0.650068998336792
Get table content time: 0.7070600986480713
Get table content time: 0.6114528179168701
Get table content time: 0.5821130275726318
Get table content time: 0.5367560386657715
Get table content time: 0.7185683250427246
Get table content time: 0.6554841995239258
Get table content time: 0.5846197605133057
Get table content time: 0.5970542430877686
Get table content time: 0.6863827705383301
Get table content time: 0.5595071315765381
Get table content time: 0.41869282722473145
